spec/fixtures/docker_provision/docker_provision.tf

```diff
@@ -0,0 +1,33 @@
+# Configure the Docker provider
+provider "docker" {}
+
+# Create n sshd servers, map internal ports
+resource "docker_container" "sshd" {
+  image = "${docker_image.sshd.latest}"
+  count = "1"
+  name  = "docker_target_${count.index}"
+  ports {
+    internal = 22
+    external = 2200 + count.index
+  }
+}
+
+resource "docker_image" "sshd" {
+  name = "rastasheep/ubuntu-sshd"
```

Comment:
I don't *think* we actually use this for anything? I think using https://hub.docker.com/_/alpine might make this a bit faster - it's 5MB as opposed to 100MB

Reply:
yeah, that will work. If we ever want an integration test that does anything over ssh we will probably want to revert back to ubuntu-sshd.

Reply:
@donoghuc It doesn't look like this actually got updated? Was that intentional? Not a big deal if you want to leave it as ubuntu, just checking it was on purpose.